packages/graphql-transformers-e2e-tests/src/testSetup.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Resolves the AWS region to run the tests
+ * @param defaultRegion 
+ * @returns 
+ */
+export const resolveTestRegion = (defaultRegion = 'us-west-2') => {
+  const resolvedRegion = process?.env?.CLI_REGION || defaultRegion;
```

Comment:
I'm not sure how this `CLI_REGION` is populated in data pipelines.
In the cli we noticed that sticking to region within one pipeline setup (i.e. all re-runs) may lead to less successful retries.
Consider a mechanism where CLI_REGION is randomly selected on each re-run from failed.

Reply:
The region is chosen randomly and populated in the test job's CI environment via the dynamically generated CI config. I like your idea of re-trying with a different region. Since that would impact all e2e tests even ones outside the transformers e2e tests, I would be inclined to do a separate enhancement PR for that.

Reply:
yep, and that stickiness across all retries is sometimes troublesome if test lands on "bad" region.